packages/@uppy/companion-client/src/RequestClient.js

```diff
@@ -23,12 +25,15 @@ module.exports = class RequestClient {
   get defaultHeaders () {
     return {
       'Accept': 'application/json',
-      'Content-Type': 'application/json'
+      'Content-Type': 'application/json',
+      'Uppy-Versions': '@uppy/companion-client=1.0.3'
```

Comment:
We’ve now added `Plugin.VERSION` property available on all plugins, including `RequestClient`, so we should send that instead of the hardcoded value.

Reply:
I'll rather change this in a subsequent PR when we are sure the `Plugin.VERSION` change is stable. And also to avoid the need to rebase. Is that ok?